src/Makefile

```diff
@@ -9,6 +9,8 @@ GOFLAGS:=-ldflags "-X main.version=$(VERSION)"
 $(BINARY_NAME): test
 	go build -o $@ $(GOFLAGS) ./cmd/cli
 
+.PHONY: protos
```

Comment:
FYI: the default rule `${BINARY_NAME)` implies `test` which implies `$(PROTOS)`

Reply:
I know, but I wanted to be able to only build the protos without running the tests